plugins/infinispan80/src/main/java/org/radargun/stream/example/WordCount.java

```diff
@@ -0,0 +1,44 @@
+package org.radargun.stream.example;
+
+import org.infinispan.stream.CacheCollectors;
+import org.radargun.traits.Streamable;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+public class WordCount implements Streamable.StreamFunction {
+
+   private Object result;
+
+   @Override
+   public Object apply(Stream stream) {
+      result = stream.map((Serializable & Function<Map.Entry<String, String>, String[]>) e -> e.getValue().split("[\\p{Punct}\\s&&[^'-]]+")).
```

Comment:
@wburns But you can't add advanced externalizer on the fly (once cache manager is built), which makes the use rather complicated. Should you add all possible externalizers to global configuration for all RG use cases?
Is the serialization cost for this lambda worth it?